cm/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "cm-integration",
+  "version": "0.0.1",
+  "description": "A server that connects Google's dialogflow to the CM.com platform",
```

Comment:
nit: Dialogflow

Reply:
Capitalized Dialogflow. 
Note: this was copied from the `package.json` file in the sibling folders. It's lowercase in other projects.